OracleManagementAgent/README.md

```diff
@@ -1,107 +1,129 @@
+<!-- markdownlint-enable MD013 -->
 ## How to build and run
 
-Oracle Management Agent image uses the official `oraclelinux:7-slim` container image as the base image.
+Oracle Management Agent image uses the official `oraclelinux:8-slim` container image as the base image.
 
-#### Prerequisites
+## Prerequisites
 
 1. [Download the Management Agent software](https://cloud.oracle.com/macs)
 
     **Note: Select 'Downloads and Keys' then download 'Agent for LINUX (X86_64)' of the package type ZIP.**
 
-1. Copy the downloaded bundle to the same directory as the Dockerfile
+1. Copy the downloaded bundle to the same directory as the `Dockerfile`
 
     ```shell
+    # stage install bundle
     $ cp oracle.mgmt_agent.zip OracleManagementAgent/dockerfiles/latest/
     ```
 
-1. Follow the steps in the [Create Install Key](https://docs.oracle.com/en-us/iaas/management-agents/doc/management-agents-administration-tasks.html#GUID-C841426A-2C32-4630-97B6-DF11F05D5712) and [Configure a Response File](https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html#GUID-5D20D4A7-616C-49EC-A994-DA383D172486) sections of the [Management Agent](https://docs.oracle.com/en-us/iaas/management-agents/index.html) documentation to create an install key and save it locally as `input.rsp`.
-
-1. Copy the downloaded install key to the same directory as the Dockerfile
+1. Change to the directory in which the `Dockerfile` for this container image is located
 
     ```shell
-    $ cp input.rsp OracleManagementAgent/dockerfiles/latest/
+    # to build and run container from this location
+    $ cd OracleManagementAgent/dockerfiles/latest/
     ```
-
-1. Change to the directory in which the `Dockerfile` for this container image is located
+<!-- markdownlint-disable MD013 -->
+1. Follow the steps in the [Create Install Key](https://docs.oracle.com/en-us/iaas/management-agents/doc/management-agents-administration-tasks.html#GUID-C841426A-2C32-4630-97B6-DF11F05D5712) and [Configure a Response File](https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html#GUID-5D20D4A7-616C-49EC-A994-DA383D172486) sections of the [Management Agent](https://docs.oracle.com/en-us/iaas/management-agents/index.html) documentation to create an install key and save it locally as `input.rsp` in the current directory.
+<!-- markdownlint-enable MD013 -->
+1. Create a bind mount location to persist container data
 
     ```shell
-    $ cd OracleManagementAgent/dockerfiles/latest/
+    # Important: cleanup any old state files
+    $ rm -rf /home/$USER/oracle
```

Comment:
The user is free to choose storage location anywhere on their system. In this example we use home because when running a container with runAsUser flag the persistent store user ownership must match with the user being used.

Reply:
Right, but `$USER` is not defined, nor do you even document how to set it. Thus, if someone just copy and pastes this command, it's going to run `rm -rf /home//oracle` which would likely be unexpected and may be catastrophic.

The big thing here is never, ever, ever document or provide an `rm -rf` example unless you're absolutely sure the rest of the example, if unmodified, would never delete anything unexpected.